pylintrc

```diff
@@ -1,5 +1,5 @@
 [MASTER]
-disable=bad-whitespace,
+disable=unnecessary-dunder-call,
```

Comment:
```
************* Module trio_util._async_itertools
src/trio_util/_async_itertools.py:5:13: C2801: Unnecessarily calls dunder method __aiter__. Use aiter built-in function. (unnecessary-dunder-call)
src/trio_util/_async_itertools.py:12:33: C2801: Unnecessarily calls dunder method __anext__. Use anext built-in function. (unnecessary-dunder-call)
************* Module trio_util._trio_async_generator
src/trio_util/_trio_async_generator.py:50:42: C2801: Unnecessarily calls dunder method __anext__. Use anext built-in function. (unnecessary-dunder-call)
```

Reply:
Rather than a blanket disable because of a warning specific to newer Python, I think it's better to tell pylint the minimum Python version that the codebase supports.  (Codebase supporting Python < 3.10 can't use `anext()`, etc.)

pylintrc:
```
# Minimum Python version to use for version dependent checks. Will default to
# the version used to run pylint.
py-version=3.7
```